src/agent/tools/expand_context.py

```
@@ -0,0 +1,318 @@
+                            error=f"Similarity expansion failed: {e}",
+                        )
+
+                elif expansion_mode == "hybrid":
+                    # Combine section + similarity
+                    section_chunks = self._expand_by_section(target_chunk, layer3_chunks, k=k // 2)
+                    expansion["expanded_chunks"].extend(section_chunks)
+
+                    try:
+                        similarity_chunks = self._expand_by_similarity(target_chunk, k=k // 2)
+                        expansion["expanded_chunks"].extend(similarity_chunks)
+                    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)